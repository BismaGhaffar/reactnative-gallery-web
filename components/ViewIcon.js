import React from 'react'
import PropTypes from 'prop-types'
import { Eye } from 'react-feather'
import { Icon, renderSmallIcon } from './Icon'

const Comment = ({ number }) => (
  <Icon>
    {renderSmallIcon(Eye)}
    <Icon.Label>{number}</Icon.Label>
  </Icon>
)

Comment.propTypes = {
  number: PropTypes.number
}

Comment.defaultProps = {
  number: 0
}
export default Comment
