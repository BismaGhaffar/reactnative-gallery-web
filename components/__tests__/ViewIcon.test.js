import React from 'react'
import renderer from 'react-test-renderer'
import ViewIcon from '../ViewIcon'

it('ViewIcon can be created', () => {
  const comp = renderer.create(<ViewIcon />)
  expect(comp).toBeDefined()
})

it('<ViewIcon /> toMatchSnapshot', () => {
  const tree = renderer.create(<ViewIcon />).toJSON()
  expect(tree).toMatchSnapshot()
})
