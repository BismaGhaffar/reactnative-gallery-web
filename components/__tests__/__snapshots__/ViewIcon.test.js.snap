// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewIcon /> toMatchSnapshot 1`] = `
<div
  className="Icon-s7tm2sn-0 cXvtWu"
>
  <svg
    fill="none"
    height={22}
    stroke="#fff"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    viewBox="0 0 24 24"
    width={22}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z"
    />
    <circle
      cx="12"
      cy="12"
      r="3"
    />
  </svg>
  <span
    className="Icon__Label-s7tm2sn-1 grNILv"
  >
    0
  </span>
</div>
`;
